app/renderer.tsx

```diff
@@ -79,6 +79,7 @@ export const applyTransform: Record<string, (val: FormDataEntryValue) => any> =
     range: parseIntFloat,
     select: (val) => (val ? JSON.parse(`${val}`) : null),
     file: (val) => (val ? JSON.parse(`${val}`) : null),
+    switch: Boolean,
```

Comment:
I believe this is not needed with the dedicated switch component?

Reply:
@devxpy It doesn't work without this, the form state going to the server do not transform the value for the switches. And I have kept it a input type checkbox because that makes things a lot easier to control rather than having a entirely custom component

Reply:
got it. thanks for inspecting!